package.json

```diff
@@ -8,12 +8,9 @@
   "devDependencies": {
     "@babel/core": "^7.16.0",
     "@babel/preset-env": "^7.16.4",
-    "@inrupt/solid-client-authn-core": "^1.12.2",
```

Comment:
Incorrect removals?

Reply:
this was done by a yarn run (idk which command unfort)! i was surprised that yarn did it and didn't expect it to be needed. i'll revert it back to how it was before.